Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -719,7 +719,7 @@ class MutableComputedValues final : public ComputedValues {
         m_inherited = static_cast<MutableComputedValues const&>(other).m_inherited;
     }
 
-    void set_aspect_ratio(AspectRatio aspect_ratio) { m_noninherited.aspect_ratio = aspect_ratio; }
+    void set_aspect_ratio(AspectRatio const& aspect_ratio) { m_noninherited.aspect_ratio = aspect_ratio; }
```

Comment:
The old code would do copy-ellision, while the new code will make a copy, which seems to be the opposite of what's intended. This is true for every line in this commit.

Reply:
I have edited it as suggested by @trflynn89